example/example-etcd-cluster-with-pv.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: "etcd.database.coreos.com/v1beta2"
+kind: "EtcdCluster"
+metadata:
+  name: "example-etcd-cluster"
+spec:
+  size: 3
+  version: "3.1.8"
+  pod:
+    usePV: true
+    pvPolicy:
+      volumeSizeInMB: 1024
```

Comment:
This is just a proposed spec. I put it under the PodPolicy but I'm not sure about its location. I'm also not sure about the `usePV` bool name and the related pvPolicy. I'm open for any better idea.